est/004.est

```diff
@@ -0,0 +1,24 @@
+date: 13-8-2016
```

Comment:
@amihaiemil we generally follow the rule that days and months are written using two digits, so 113-8-20161 should be 113-08-20161